AndroidP/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
+.externalNativeBuild
```

Comment:
You sure this is needed? You may want to look at [this](https://github.com/novoda/spikes/blob/54b73a7ebe60aa296c9cbe76d64b49783781340d/sliceanddice/.gitignore) `.gitignore` instead